tests/commands/functions.py

```diff
@@ -0,0 +1,19 @@
+"""
+`functions` command test module
+"""
+
+
+from tests.utils import gdb_run_cmd
+from tests.utils import GefUnitTestGeneric
+
+
+class FunctionsCommand(GefUnitTestGeneric):
+    """`functions` command test module"""
+
+
+    def test_cmd_functions(self):
+        cmd = "functions"
+        res = gdb_run_cmd(cmd)
+        self.assertNoException(res)
+        self.assertIn("$_heap", res)
```

Comment:
let's test for them all? `_base`, `_bss`, `_got`, `_heap`, `_stack`

Reply:
that's what was already there
https://github.com/hugsy/gef/blob/283690ae9bfcecbb3deb80cd275d327c46b276b5/tests/runtests.py#L191